platform/util/platform-util/pom.xml

```diff
@@ -70,6 +70,12 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
+            <artifactId>system-rules</artifactId>
+            <version>1.16.1</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
❓ Do we use this external dependency elsewhere? I know we're usually pretty leery about this sort of thing.